packages/lab/src/tabs-next/TabstripNext.tsx

```diff
@@ -0,0 +1,178 @@
+import { makePrefixer, useControlled } from "@salt-ds/core";
+import clsx from "clsx";
+import {
+  Children,
+  cloneElement,
+  isValidElement,
+  KeyboardEvent,
+  MouseEvent,
+  PropsWithChildren,
+  ReactNode,
+  useRef,
+  useState,
+} from "react";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { Overflow, OverflowItem } from "@fluentui/react-overflow";
+import { TabNext } from "./TabNext";
+import { TabElement, TabProps } from "./TabsNextTypes";
+import { OverflowMenu } from "./OverflowMenu";
+import tabstripCss from "./TabstripNext.css";
+
+const withBaseName = makePrefixer("saltTabstripNext");
+
+function isTab(child: ReactNode | TabElement): child is TabElement {
+  return isValidElement(child) && child.type === TabNext;
+}
+
+export type TabstripNextProps = PropsWithChildren<{
+  /* Value for the uncontrolled version. Set to null in order to have no tabs be selected. */
+  selectedTab?: string | null;
+  /* Callback for the controlled version. */
+  onSelectTab?: (
+    e: KeyboardEvent<HTMLElement> | MouseEvent<HTMLElement>,
+    value: string
+  ) => void;
+  /* Initial value for the uncontrolled version. Set to null in order to have no tabs be selected by default. */
+  defaultSelectedTab?: string | null;
+  /* Align the tabs to the center. Left aligned by default. */
+  align?: "center";
```

Comment:
Type should be 2?

Reply:
Yeah, it should probably be left and center. Instead of undefined and center.